Fastlane/deployment_lanes.rb

```diff
@@ -33,72 +33,75 @@
+  if branch_name.nil? || branch_name.empty?
+    branch_name = git_branch
+  end
 
-    puts "Created release with URL: #{release_url}"
-
-    begin
-      testflight(
-        beta_app_review_info: {
-          contact_email: options[:contact_email] || ENV['BETA_CONTACT_EMAIL'],
-          contact_first_name: options[:contact_first_name] || ENV['BETA_CONTACT_FIRST_NAME'],
-          contact_last_name: options[:contact_last_name] || ENV['BETA_CONTACT_LAST_NAME'],
-          contact_phone: options[:contact_phone] || ENV['BETA_CONTACT_PHONE'],
-          demo_account_name: options[:demo_account_name] || ENV['BETA_DEMO_ACCOUNT_NAME'],
-          demo_account_password: options[:demo_account_password] || ENV['BETA_DEMO_ACCOUNT_PASSWORD']
-        },
-        skip_waiting_for_build_processing: false,
-        skip_submission: false,
-        groups: options[:groups] || ENV['TESTFLIGHT_GROUPS_BETA'],
-        changelog: stripped_changelog,
-        team_id: options[:team_id] || ENV['FASTLANE_ITC_TEAM_ID']
-      )
-    rescue StandardError => e
-      raise e unless e.message.include?('Another build is in review')
+  # Create a new GitHub release
+  last_non_candidate_tag = latest_github_non_candidate_tag
+  release_title = "#{tag_name} - App Store Release Candidate"
+  release_output = sh("mint run --silent gitbuddy release -l #{last_non_candidate_tag} -b develop --skip-comments --json --use-pre-release --target-commitish #{branch_name} --tag-name #{tag_name} --release-title '#{release_title}'")
+  release_json = JSON.parse(release_output)
 
-      UI.important('TestFlight delivery failed because a build is already in review, but continuing anyway!')
-    end
+  release_url = release_json['url']
+  changelog = release_json['changelog']
+  stripped_changelog = strip_markdown_url(input: changelog)
+
+  UI.message "Created release with URL: #{release_url}"
+
+  begin
+    testflight(
+      beta_app_review_info: {
+        contact_email: options[:contact_email] || ENV['BETA_CONTACT_EMAIL'],
+        contact_first_name: options[:contact_first_name] || ENV['BETA_CONTACT_FIRST_NAME'],
+        contact_last_name: options[:contact_last_name] || ENV['BETA_CONTACT_LAST_NAME'],
+        contact_phone: options[:contact_phone] || ENV['BETA_CONTACT_PHONE'],
+        demo_account_name: options[:demo_account_name] || ENV['BETA_DEMO_ACCOUNT_NAME'],
+        demo_account_password: options[:demo_account_password] || ENV['BETA_DEMO_ACCOUNT_PASSWORD']
+      },
+      groups: options[:groups] || ENV['TESTFLIGHT_GROUPS_BETA'],
+      changelog: stripped_changelog,
+      team_id: options[:team_id] || ENV['FASTLANE_ITC_TEAM_ID']
+    )
+  rescue StandardError => e
+    raise e unless e.message.include?('Another build is in review')
 
-    slack_message(message: 'A new Release Candidate has been published.', tag_name: tag_name, release_url: release_url)
+    UI.important "TestFlight delivery failed because a build is already in review, but continuing anyway!"
```

Comment:
Heeeey, I love this! We actually have [this issue](https://wetransfer.atlassian.net/browse/TMB-935?atlOrigin=eyJpIjoiMzAzODY4NGIzOGEyNGMxYzk1YjdlYjQ5MDYyZDNjN2QiLCJwIjoiaiJ9) open to work on. Would it make sense for you to pick that up while you're in the right context either way?